cmd/buildah/config.go

```diff
@@ -18,58 +18,58 @@ const (
 
 var (
 	configFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "annotation, a",
+			Usage: "add `annotation` e.g. annotation=value, for the target image",
+		},
 		cli.StringFlag{
-			Name:  "author",
-			Usage: "image author contact `information`",
+			Name:  "arch",
+			Usage: "set `architecture` of the target image",
 		},
 		cli.StringFlag{
-			Name:  "created-by",
-			Usage: "`description` of how the image was created",
-			Value: DefaultCreatedBy,
+			Name:  "author",
+			Usage: "set image author contact `information`",
 		},
 		cli.StringFlag{
-			Name:  "arch",
-			Usage: "`architecture` of the target image",
+			Name:  "cmd",
+			Usage: "Sets the default `command` to run for containers based on the image",
 		},
 		cli.StringFlag{
-			Name:  "os",
-			Usage: "`operating system` of the target image",
+			Name:  "created-by",
+			Usage: "add `description` of how the image was created",
+			Value: DefaultCreatedBy,
 		},
 		cli.StringFlag{
-			Name:  "user, u",
-			Usage: "`user` to run containers based on image as",
+			Name:  "entrypoint",
+			Usage: "set `entry point` for containers based on image",
 		},
 		cli.StringSliceFlag{
-			Name:  "port, p",
-			Usage: "`port` to expose when running containers based on image",
+			Name:  "env, e",
+			Usage: "add `environment variable` to be set when running containers based on image",
 		},
 		cli.StringSliceFlag{
-			Name:  "env, e",
-			Usage: "`environment variable` to set when running containers based on image",
+			Name:  "label, l",
+			Usage: "add image configuration `label` e.g. label=value",
 		},
 		cli.StringFlag{
-			Name:  "entrypoint",
-			Usage: "`entry point` for containers based on image",
+			Name:  "os",
+			Usage: "set `operating system` of the target image",
+		},
+		cli.StringSliceFlag{
+			Name:  "port, p",
+			Usage: "add `port` to expose when running containers based on image",
 		},
 		cli.StringFlag{
-			Name:  "cmd",
-			Usage: "`command` for containers based on image",
+			Name:  "user, u",
+			Usage: "set `user` to run containers based on image as",
 		},
 		cli.StringSliceFlag{
 			Name:  "volume, v",
-			Usage: "`volume` to create for containers based on image",
+			Usage: "add `volume` to create for containers based on image",
```

Comment:
I'd personally vote to leave this as is, or at least not use 'add'.  Perhaps 'set'  if you feel the need?  The manpage has: 
```
--volume volume

Specifies a location in the directory tree which should be marked as a volume in any images which will be built using the specified container.
```
Which I'm not crazy about, but we're not really adding or creating,

Reply:
My idea was to use "add" when you can call it more then once, and `set` when you calling it once is all that makes sense.  Since you can have multiple volumes, I do add, but if you are useing working_dir, it can only be `set`.  But we are bike shedding.

Reply:
Your call.  Truth be told I was back/forth on this as well and thought set was the better option.

Reply:
How about?
			Usage: "add builtin `volume` path to be created for containers based on image",

Reply:
I like!